ajax/libs/post-robot/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "post-robot",
+  "version": "7.0.2",
+  "filename": "post-robot.min.js",
+  "description": "Simple postMessage based server.",
+  "keywords": [
+    "cross-domain",
+    "cross domain",
+    "xdm",
+    "iframe",
+    "postmessage",
+    "krakenjs",
+    "kraken"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/krakenjs/post-robot.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Daniel Brain <@bluepnume>",
```

Comment:
Please modify this as `Daniel Brain <dbrain@paypal.com>`
Ref: https://github.com/krakenjs/post-robot/blob/master/bower.json#L6